chapter_appendix_math/statistics_vn.md

```diff
@@ -619,13 +626,16 @@ Rather, it would be far better if we could produce an interval that would contai
 If you were interested in such ideas a century ago, then you would have been excited to read "Outline of a Theory of Statistical Estimation Based on the Classical Theory of Probability" by Jerzy Neyman :cite:`Neyman.1937`, who first introduced the concept of confidence interval in 1937.
 -->
 
-*dịch đoạn phía trên*
+Khi ước lượng giá trị của tham số $\theta$, bộ ước lượng điểm như $\hat \theta$ bị hạn chế vì chúng không bao hàm sự bất định.
+Thay vào đó, sẽ tốt hơn nhiều nếu ta có thể tìm ra một khoảng chứa tham số $\theta$ thực sự với xác suất cao.
```

Comment:
```suggestion
Thay vào đó, nếu ta có thể tìm ra một khoảng chứa tham số $\theta$ thật sự với xác suất cao sẽ tốt hơn nhiều.
```

Reply:
Từ `thật sự` và `thực sự` trên căn bản không có gì khác nhau. Tuy nhiên, tuỳ người mà cách sử dụng khác nhau. Mình đề xuất sử dụng từ `thật sự` và thấy cần thống nhất sử dụng một từ cho toàn văn bản.